### PR TITLE
Fix CodeNarc IP violations

### PR DESCRIPTION
**Pull request by asodja**
By resolving root project dir via `IsolatedProject` instead of `Project`.

### CODE REVIEW

**Comment by asodja** [comment]
@bot-gradle test this

**Comment by asodja** [comment]
@bot-gradle test this

**Comment by asodja** [comment]
@bot-gradle cancel

**Comment by asodja** [comment]
@bot-gradle test this

**Comment by asodja** [comment]
@bot-gradle test this

File: platforms/jvm/code-quality/src/main/groovy/org/gradle/api/plugins/quality/CodeNarcPlugin.java [hunk 1]
```diff
@@ -48,8 +48,10 @@
 public abstract class CodeNarcPlugin extends AbstractCodeQualityPlugin<CodeNarc> {
 
     public static final String DEFAULT_CODENARC_VERSION = appropriateCodeNarcVersion();
+    private static final String CONFIG_FILE_PATH = "config/codenarc/codenarc.xml";
```
Comment (alllex) [inline platforms/jvm/code-quality/src/main/groovy/org/gradle/api/plugins/quality/CodeNarcPlugin.java:51]: 💅 Would it make sense to prefix it with `DEFAULT_`?

**Review by alllex — APPROVED** [review]
LGTM!

File: platforms/jvm/code-quality/src/main/groovy/org/gradle/api/plugins/quality/CodeNarcPlugin.java (hunk 1 above)
Comment (asodja) [reply platforms/jvm/code-quality/src/main/groovy/org/gradle/api/plugins/quality/CodeNarcPlugin.java]: Makes sense yes, fixed

**Pull request merged**